test/homepage-noweb-test.js

```diff
@@ -41,24 +40,31 @@ suite.addBatch({
         },
         "and we visit the root URL": {
             topic: function() {
-                var browser;
-                browser = new Browser();
+                var cb=this.callback,
+                    browser = new Browser();
 
-                browser.visit("http://localhost:4815/", this.callback);
+                browser.visit("http://localhost:4815/", function(){
+                        cb(!browser.success, browser);
```

Comment:
True, this is not the best way. My focus was really at making the test work with the minimal amount of changes. But I wanted to make the PR, so I would be out there. To prevent blind parallel solving stuff. (not my best English). The correct way would be to get rid of the `err` in the callback, I have to look at the zombie api and examples again. Later more on this. Need to catch some sleep.